src/main/java/dao/UserProfileDao.java

```diff
@@ -0,0 +1,42 @@
+package dao;
+
+import entity.UserProfile;
+import dbService.executor.Executor;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+public class UserProfileDao implements Dao<UserProfile> {
+
+    private Executor executor;
+
+    public UserProfileDao(Connection connection) {
+        this.executor = new Executor(connection);
+    }
+
+    @Override
+    public UserProfile get(String login) throws SQLException {
+            return executor.execQuery("SELECT * FROM users WHERE login='" + login + "'",
+                    resultSet -> {
+                        resultSet.next();
+                        return new UserProfile(resultSet.getString(2), resultSet.getString(3));
+                    });
+    }
+
+    public Long getIdByName(String name) throws SQLException {
+        return executor.execQuery("select * from users where login='" + name + "'", resultSet -> {
```

Comment:
```suggestion
        return executor.execQuery("SELECT * FROM users WHERE login='" + name + "'", resultSet -> {
```
the same in all query